extension/substrait/CMakeLists.txt

```diff
@@ -0,0 +1,81 @@
+    ../../third_party/google/protobuf/generated_enum_util.cc
+    ../../third_party/google/protobuf/generated_message_bases.cc
+    ../../third_party/google/protobuf/generated_message_reflection.cc
+    ../../third_party/google/protobuf/generated_message_table_driven.cc
+    ../../third_party/google/protobuf/generated_message_table_driven_lite.cc
+    ../../third_party/google/protobuf/generated_message_util.cc
+    ../../third_party/google/protobuf/implicit_weak_message.cc
+    ../../third_party/google/protobuf/inlined_string_field.cc
+    ../../third_party/google/protobuf/map.cc
+    ../../third_party/google/protobuf/map_field.cc
+    ../../third_party/google/protobuf/message.cc
+    ../../third_party/google/protobuf/message_lite.cc
+    ../../third_party/google/protobuf/parse_context.cc
+    ../../third_party/google/protobuf/port_def.inc
+    ../../third_party/google/protobuf/port_undef.inc
+    ../../third_party/google/protobuf/reflection_ops.cc
+    ../../third_party/google/protobuf/repeated_field.cc
+    ../../third_party/google/protobuf/repeated_ptr_field.cc
+    ../../third_party/google/protobuf/text_format.cc
+    ../../third_party/google/protobuf/unknown_field_set.cc
+    ../../third_party/google/protobuf/wire_format.cc
+    ../../third_party/google/protobuf/wire_format_lite.cc
+    ../../third_party/google/protobuf/io/coded_stream.cc
+    ../../third_party/google/protobuf/io/io_win32.cc
+    ../../third_party/google/protobuf/io/strtod.cc
+    ../../third_party/google/protobuf/io/tokenizer.cc
+    ../../third_party/google/protobuf/io/zero_copy_stream.cc
+    ../../third_party/google/protobuf/io/zero_copy_stream_impl.cc
+    ../../third_party/google/protobuf/io/zero_copy_stream_impl_lite.cc
+    ../../third_party/google/protobuf/stubs/common.cc
+    ../../third_party/google/protobuf/stubs/int128.cc
+    ../../third_party/google/protobuf/stubs/status.cc
+    ../../third_party/google/protobuf/stubs/stringpiece.cc
+    ../../third_party/google/protobuf/stubs/stringprintf.cc
+    ../../third_party/google/protobuf/stubs/structurally_valid.cc
+    ../../third_party/google/protobuf/stubs/strutil.cc
+    ../../third_party/google/protobuf/stubs/substitute.cc)
+
+set(SUBSTRAIT_SOURCES
+    ../../third_party/substrait/substrait/capabilities.pb.cc
+    ../../third_party/substrait/substrait/expression.pb.cc
+    ../../third_party/substrait/substrait/function.pb.cc
+    ../../third_party/substrait/substrait/parameterized_types.pb.cc
+    ../../third_party/substrait/substrait/plan.pb.cc
+    ../../third_party/substrait/substrait/relations.pb.cc
+    ../../third_party/substrait/substrait/type.pb.cc
+    ../../third_party/substrait/substrait/type_expressions.pb.cc
+    ../../third_party/substrait/substrait/extensions/extensions.pb.cc)
+
+add_library(
+  substrait_extension STATIC
+  to_substrait.cpp from_substrait.cpp substrait-extension.cpp
+  ${SUBSTRAIT_SOURCES} ${PROTOBUF_SOURCES})
+
+build_loadable_extension(substrait substrait-extension.cpp)
```

Comment:
This is missing many sources, it will fail when actually used as a loadable extension (although this is not tested right now).